packages/cli/src/commands/execute-batch.ts

```diff
@@ -263,6 +264,11 @@ export class ExecuteBatch extends BaseCommand {
 			ExecuteBatch.githubWorkflow = true;
 		}
 
+		if (this.globalConfig.taskRunners.enabled) {
```

Comment:
there are at least 4 commands that do this now. maybe we should create a new flag in the `BaseCommand` similar to `needsCommunityPackages`, instead of duplicating this in 4 places?